### DISCUSSION TITLE
Cartographer install

### DISCUSSION THREAD

**Discussion by golia28** [Guilouz/Creality-Helper-Script-Wiki#726] [Features Request]
Should the cartographer be configured using the helper script? Or without helper script but with the klipper installed through helper script?

Thanks

**Comment by DreadPirateRobertz** [comment]
## Test Coverage Convoy — Status Update (2026-03-14)

Mayor directed **Option 3: Test coverage expansion** across the crew. Here's the result:

### My batch (miquella):
- All 5 assigned backend files already had tests — crew delivered them earlier this session
- Pivoted to **4 untested public helpers**: DeliveryEstimator (22 tests), elementIdValidator (23 tests), faqSeo (7 tests), localBusinessSeo (8 tests)
- **PR #317** — 72 new tests, 5-agent review running

### Radahn's batch:
- **PR #316** — 4 backend service modules, 378 tests, 5-agent review complete

### Remaining gaps:
- **14 page files** untested (5,818 lines) — all \$w-heavy wiring, lower ROI for unit tests
- 0 untested backend files, 0 untested public helpers remaining

Suite: **12,603 tests, 326 files** (my branch). The test coverage convoy is a good pattern for idle-time work.

**Reply by thefunkygibbon** [reply]
what??
